### ISSUE TITLE
New Buildstream release

### ISSUE THREAD

**Issue by Tanvir786123** [apache/buildstream#1871]
### Context

When building the `freedesktop-sdk.bst:components/cryptsetup-base.bst`, we receive the following compile errors:
```
    touch stamp-po
    make[2]: Leaving directory '/buildstream-build/bst_build_dir/po'
    Making all in tests
    make[2]: Entering directory '/buildstream-build/bst_build_dir/tests'
    ../../tests/generate-symbols-list ../../lib/libcryptsetup.sym > ./test-symbols-list.h
    ../../tests/generate-symbols-list: line 9: /dev/fd/63: No such file or directory
    make[2]: *** [Makefile:1204: test-symbols-list.h] Error 1
    make[2]: Leaving directory '/buildstream-build/bst_build_dir/tests'
    make[1]: *** [Makefile:2685: all-recursive] Error 1
    make[1]: Leaving directory '/buildstream-build/bst_build_dir'
    make: *** [Makefile:1302: all] Error 2
    Command 'if [ -n "bst_build_dir" ]; then
      cd bst_build_dir
    fi
    make ' failed with exitcode 2
    [00:00:17] FAILURE freedesktop-sdk.bst:components/cryptsetup-base.bst: Running commands
    [--:--:--] START   [206a1e6e] freedesktop-sdk.bst:components/cryptsetup-base.bst: Caching artifact
    [00:00:00] FAILURE [206a1e6e] freedesktop-sdk.bst:components/cryptsetup-base.bst: Command failed
```

This is **virtually identical** to [this issue](https://gitlab.com/cryptsetup/cryptsetup/-/issues/829).

### Description

I have been told that we need a new Buildstream release so that the bundle is updated.

### Acceptance Criteria

We provide evidence that `freedesktop-sdk.bst:components/cryptsetup-base.bst` compiles correctly

CC:
- @jjardon
- @abderrahim
- @nanonyme

**Comment by nanonyme** [comment]
The problem here is BuildStream package bundles buildbox rather than latter having freedom of being released when needed. There has now been overlong period since last refresh of buildbox components in BuildStream wheels and it's resulting in breakage

**Comment by Tanvir786123** [comment]
I have updated the title and description to be more accurate @jjardon

**Comment by gtristan** [comment]
We already have an issue for de-prioritizing the static buildbox binaries included in the complimentary wheels we generate (along side our official tarball releases) for added convenience.

I'll close this given that we have today released 2.1.0.

**Comment by Tanvir786123** [comment]
> We already have an issue for de-prioritizing the static buildbox binaries included in the complimentary wheels we generate (along side our official tarball releases) for added convenience.
> 
> I'll close this given that we have today released 2.1.0.

Can indeed confirm that the new release has enabled me to compile `cryptsetup.bst`.

Thank you :)
